<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AustrianMeanElevationDialogBase</class>
 <widget class="QDialog" name="AustrianMeanElevationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gpsinfo4zemokost</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>8</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer auswählen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectLayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="onlySelFeat">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>34</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nur ausgewählte Features</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>340</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
   <property name="icon">
    <iconset theme="window-close">
     <normaloff/>
    </iconset>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>340</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> Hilfe</string>
   </property>
   <property name="icon">
    <iconset theme="help-contents">
     <normaloff/>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> About</string>
   </property>
   <property name="icon">
    <iconset theme="dialog-information">
     <normaloff/>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>340</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> Speichern</string>
   </property>
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>581</width>
     <height>211</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
